packages/devtools_app/lib/src/shared/config_specific/notifications/_notifications_web.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2023 The Chromium Authors. All rights reserved.
```

Reply:
this was just a rename of the file, so will leave the imports as is.